radio/util/hw_defs/legacy_names.py

```diff
@@ -2,7 +2,7 @@
 
 LEGACY_NAMES = [
     {
-        "targets": {"x10", "x10express", "t16", "tx16s", "t18"},
+        "targets": {"x10", "x10express", "t16", "tx16s", "t18","f16"},
```

Comment:
For consistency:
```suggestion
        "targets": {"x10", "x10express", "t16", "tx16s", "t18", "f16"},
```

Looking forward to seeing what this handset is like...